workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowRunsTabContent.tsx

```diff
@@ -202,7 +153,67 @@ export const WorkflowRunsTabContent = () => {
     ),
     [statusSelectorValue],
   );
-
+  
+  const applyBackendSort = React.useCallback((item1: WorkflowRunDetail, item2: WorkflowRunDetail): number => {          
+    // Workaround for material-table applying sorting on top of backend sorting. The version we are using is too old to request a fix. 
+    // Should be resolved when upgrading backstage and all plugins to material6
```

Comment:
create a jira for this as well and add the link in this comment.
 I'd like to avoid to forget fixing this.

Reply:
I'm not sure there's a point having a Jira issue opened in the backlog that is not solvable until upgrade to material6. I can turn this comment into a TODO.